### PR TITLE
add exporter/tanzuobservabilityexporter

### PR DESCRIPTION
**Pull request by oppegard**
**Description:**
This change adds an exporter for distributed traces in [Tanzu Observability (Wavefront)](https://tanzu.vmware.com/observability).

**Testing:**
Unit tests.

**Documentation:**
README added.

For questions related to this submission, please mention @thepeterstone or @oppegard.

### CODE REVIEW

File: exporter/tanzuobservabilityexporter/exporter.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	LabelError         = "error"
+	LabelEventName     = "name"
+	LabelService       = "service"
+	LabelSpanKind      = "span.kind"
+	LabelStatusMessage = "status.message"
+	LabelStatusCode    = "status.code"
+)
+
+// Sender Interface for sending tracing spans to Tanzu Observability
+type Sender interface {
```
Comment (bogdandrutu) [inline exporter/tanzuobservabilityexporter/exporter.go:43]: Does this need to be public?

**Review by bogdandrutu — CHANGES_REQUESTED** [review]
Consider to split this PR based on https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#when-adding-a-new-component

**Comment by thepeterstone** [comment]
> Consider to split this PR based on https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#when-adding-a-new-component

We are splitting this up - first PR with just structure of our component is #3393

File: exporter/tanzuobservabilityexporter/exporter.go (hunk 1 above)
Comment (oppegard) [reply exporter/tanzuobservabilityexporter/exporter.go]: Nope! We made this private and renamed to `spanSender` in #3422.

**Pull request merged**